### DISCUSSION TITLE
Potential bug: 1.11.7 generates error on deleting nodes

### DISCUSSION THREAD

**Discussion by doctomlug** [freeplane/freeplane#1380] [Help]
[log.zip](https://github.com/freeplane/freeplane/files/12520256/log.zip)

- created new map
- added two node (abc, def)
- deleted first node (del key)

I deleted the log file before generating the new map. Similar behavior is observed in "old" maps
Identical behavior on two different computers (send error report from the other one)

Edit: Tried portable installs 1.11.6 and 1.11.7 (no scripts, addons etc), portable recommended JRE.  
=> .6 runs as expected; .7 recreates the error above (reported via internal method)
Other menu activities (e.g. exporting branch to new map) generate the error as well

**Comment by dpolivaev** [comment]
Thanks for reporting, it is just a bug.

**Reply by dpolivaev** [reply]
https://github.com/freeplane/freeplane/issues/1381

**Comment by dpolivaev** [comment]
I am sorry, I just accidentally removed a not operator on one condition.
I hope it works better now.

Preview 1.11.7_02
https://sourceforge.net/projects/freeplane/files/freeplane%20preview/

**Reply by doctomlug** [reply]
Just did some tests with 1.11.7_02. It's looking good so far. will keep using it on my privat machine with error reporting on (just in case :) )
Thanks a lot for your amazing support
